features/netsocket/emac-drivers/TARGET_Freescale_EMAC/kinetis_emac.cpp

```diff
@@ -262,6 +263,77 @@ bool Kinetis_EMAC::low_level_init_successful()
     return true;
 }
 
+status_t Kinetis_EMAC::init_enet_phy(ENET_Type *base, uint32_t phyAddr, uint32_t srcClock_Hz)
+{
+    status_t result = kStatus_Success;
+    uint32_t instance = ENET_GetInstance(base);
+
+#if TARGET_K66F
```

Comment:
Looks like we would like to split the PHY_Init function into 2. Can this be done inside fsl_phy.c and then call those functions rather than move the implementation from fsl_phy.c to inside this driver.

Reply:
Yes, that sounds good. Will you make a commit about the fsl_phy.c changes? Or should I make those?

Reply:
I have attached a patch, can you take a look and update PR.

Reply:
Thanks, I'll apply the patch.